app/experimenter/experiments/tests/test_models.py

```diff
@@ -1260,6 +1260,33 @@ def test_is_ready_to_launch_is_false_without_conditional_review(self):
 
         self.assertFalse(experiment.is_ready_to_launch)
 
+    def test_review_order_is_correct_for_experiment(self):
+        experiment = ExperimentFactory.create()
```

Comment:
I'd be explicit about the experiment type here.

Reply:
One thing we see a bunch is tests where we rely on the defaults of the factory to be valid for the test then we change those defaults and a test fails that shouldn't, so we should try to always be explicit about the required values of a factory for a given test.  

I think if we go ahead with that change to make the factories more random that'll expose a lot more of these cases.